server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -75,6 +97,7 @@ public static class Names {
         public static final String SEARCH_COORDINATION = "search_coordination";
         public static final String AUTO_COMPLETE = "auto_complete";
         public static final String SEARCH_THROTTLED = "search_throttled";
+        /** Cluster management tasks. Tasks that manage data, and tasks that report on the cluster health via statistics etc. */
```

Comment:
Might be worth mentioning that this threadpool is of limited size and some of its tasks do IO or are otherwise quite heavyweight, so not to use it for latency-sensitive operations.